library/src/index.js

```diff
@@ -137,6 +137,7 @@ const Components = {
 	FontAwesomeIcon
 }
 
+Vue.component("NioError", NioError)
```

Comment:
We might want a more descriptive component name to describe this component. I think NioError could represent many things in principle. Maybe we could use NioApiError or NioApiErrorPage instead? We should discuss